examples/Commerical/basic_mgt_spoke/parameters.tfvars

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+###########################
+## Global Configuration  ##
+###########################
+
+# The prefixes to use for all resources in this deployment
+org_name           = "anoa"   # This Prefix will be used on most deployed resources.  10 Characters max.
+deploy_environment = "dev"    # dev | test | prod
+environment        = "usgovernment" # public | usgovernment
```

Comment:
This example for Commercial is targeting Gov.

Reply:
changed